src/System.Runtime.Intrinsics.X86/src/System/Runtime/Intrinsics/X86/Avx.cs

```diff
@@ -0,0 +1,577 @@
+        }
+        // __m256i _mm256_set_epi16 (short e15, short e14, short e13, short e12, short e11, short e10, short e9, short e8, short e7, short e6, short e5, short e4, short e3, short e2, short e1, short e0)
+        public static Vector256<ushort> Set(ushort e15, ushort e14, ushort e13, ushort e12, ushort e11, ushort e10, ushort e9, ushort e8, ushort e7, ushort e6, ushort e5, ushort e4, ushort e3, ushort e2, ushort e1, ushort e0)
+        {
+            throw new NotImplementedException();
+        }
+        // __m256i _mm256_set_epi32 (int e7, int e6, int e5, int e4, int e3, int e2, int e1, int e0)
+        public static Vector256<int> Set(int e7, int e6, int e5, int e4, int e3, int e2, int e1, int e0)
+        {
+            throw new NotImplementedException();
+        }
+        // __m256i _mm256_set_epi32 (int e7, int e6, int e5, int e4, int e3, int e2, int e1, int e0)
+        public static Vector256<uint> Set(uint e7, uint e6, uint e5, uint e4, uint e3, uint e2, uint e1, uint e0)
+        {
+            throw new NotImplementedException();
+        }
+        // __m256i _mm256_set_epi64x (__int64 e3, __int64 e2, __int64 e1, __int64 e0)
+        public static Vector256<long> Set(long e3, long e2, long e1, long e0) { throw new NotImplementedException(); }
+        // __m256i _mm256_set_epi64x (__int64 e3, __int64 e2, __int64 e1, __int64 e0)
+        public static Vector256<ulong> Set(ulong e3, ulong e2, ulong e1, ulong e0) { throw new NotImplementedException(); }
+        // __m256 _mm256_set_ps (float e7, float e6, float e5, float e4, float e3, float e2, float e1, float e0)
+        public static Vector256<float> Set(float e7, float e6, float e5, float e4, float e3, float e2, float e1, float e0)
+        {
+            throw new NotImplementedException();
+        }
+        // __m256d _mm256_set_pd (double e3, double e2, double e1, double e0)
+        public static Vector256<double> Set(double e3, double e2, double e1, double e0) { throw new NotImplementedException(); }
+        
+        // __m256i _mm256_set1_epi8 (char a)
+        // __m256i _mm256_set1_epi16 (short a)
+        // __m256i _mm256_set1_epi32 (int a)
+        // __m256i _mm256_set1_epi64x (long long a)
+        // __m256 _mm256_set1_ps (float a)
+        // __m256d _mm256_set1_pd (double a)
+        public static Vector256<T> Set1<T>(T value) where T : struct { throw new NotImplementedException(); }
+
+        // __m256 _mm256_set_m128 (__m128 hi, __m128 lo)
+        // __m256d _mm256_set_m128d (__m128d hi, __m128d lo)
+        // __m256i _mm256_set_m128i (__m128i hi, __m128i lo)
+        public static Vector256<T> SetHiLo<T>(Vector128<T> hi, Vector128<T> lo) where T : struct { throw new NotImplementedException(); }
+
+        // __m256i _mm256_setzero_si256 (void)
+        // __m256 _mm256_setzero_ps (void)
+        // __m256d _mm256_setzero_pd (void)
+        public static Vector256<T> SetZero<T>() where T : struct { throw new NotImplementedException(); }
+
```

Comment:
Isn't that just a static getter in the type then rather than a function?
```csharp
public struct Vector256<T> where T : struct
{
    public static Vector256<T> Zero { get { throw null; } }
}
```